File: Capstone_Project/src/main/java/capstone_project/models/Location.java
package capstone_project.models;

public class Location {

    private int locationId;
    private String address;
    private String city;
    private String state;
    private String zipCode;
    private String addressType;

    public int getLocationId() {
        return locationId;
    }

    public void setLocationId(int locationId) {
        this.locationId = locationId;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getState() {
        return state;
    }

    public void setState(String state) {
        this.state = state;
    }

    public String getZipCode() {
        return zipCode;
    }

    public void setZipCode(String zipCode) {
        this.zipCode = zipCode;
    }


    public String getAddressType() {
        return addressType;
    }

    public void setAddressType(String addressType) {
        this.addressType = addressType;
    }
}

File: Capstone_Project/client/src/components/Location.js
import Map from "./Map";
function Location ({ location }) {
  
    return (
      <>
      <tr>
        <td>{location.address}</td>
        <td>{location.city}</td>
        <td>{location.state}</td>
        <td>{location.zipCode}</td>
        <td>{location.addressType}</td>
      </tr>
      <Map address={location.address} state={location.state} city={location.city} /> 
      </>
    );

}

export default Location;